install/export-go-path-linux.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+if [[ -n "$1" ]]
+then
+  DIR=$1;
+else
+  DIR=~;
+fi
+
+if [[ -n "$2" ]]
+then
+  PROFILE=$2;
+else
+  PROFILE=~/.bash_profile;
+fi
+
+echo -e "\n\nStoring GOPATH in $PROFILE\n";
+
+(echo ""; echo 'export PATH=$PATH:/usr/local/go/bin') >> $PROFILE;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/manual/akotlar/bystro/pullRequest?prid=4888220)